.github/workflows/ci.yml

```diff
@@ -167,6 +173,7 @@ jobs:
 
   site-integration-tests:
     name: "Site integration tests"
+    if: ${{ github.event_name == 'pull_request' }}
```

Comment:
This line was added by mistake – it skips integration tests on `main`: https://github.com/blockprotocol/blockprotocol/actions/runs/2712761051

TODO: Remove in any follow-up PR

Reply:
Fixed in https://github.com/blockprotocol/blockprotocol/pull/446#discussion_r926822548